components/datadog/apps/cpustress/ecs.go

```diff
@@ -14,7 +14,7 @@ type EcsComponent struct {
 }
 
 func EcsAppDefinition(e aws.Environment, clusterArn pulumi.StringInput, opts ...pulumi.ResourceOption) (*ecsComp.Workload, error) {
-	namer := e.Namer.WithPrefix("cpustress")
+	namer := e.Namer.WithPrefix("cpustress").WithPrefix("ec2")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why the `ec2` prefix? To differentiate between ecs ec2 and ecs fargate?